packages/astro/test/benchmark/simple/server.mjs

```diff
@@ -0,0 +1,18 @@
+import http from 'http';
+import { handler } from './dist/server/entry.mjs';
+
+const listener = (req, res) => {
+	handler(req, res, err => {
+		if(err) {
+			res.writeHead(500);
+			res.end(err.toString());
+		} else {
+			res.writeHead(404);
+			res.end('Not found');
+		}
+	});
+};
+
+const server = http.createServer(listener);
+server.listen(3002);
+console.log(`Listening at http://localhost:3002`);
```

Comment:
Can we somehow ignore these ESLint warnings?
![grafik](https://user-images.githubusercontent.com/10373452/193614312-f6bea749-ffc0-4557-a619-c92db6ed9893.png)

Reply:
Yep, will do that.